fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BrokerLoadJob.java

```diff
@@ -387,4 +389,9 @@ public void afterVisible(TransactionState txnState, boolean txnOperated) {
         super.afterVisible(txnState, txnOperated);
         writeProfile();
     }
+
+    @Override
+    protected String getResourceName() {
+        return cluster != null ? cluster : "N/A";
+    }
```

Comment:
And another way that I think may be tricky is to imitate SparkLoadJob that we just use the broker name. What's ur opinion?

Reply:
As for the original `cluster` field in LoadStmt, do u it seems a little redundant?